### ISSUE TITLE
CocoaPods

### ISSUE THREAD

**Issue by lfarah** [SwiftStudies/SmarkDown#2]
I'd be cool to also add CocoaPods support. I done this implementation myself last month for my library. Just follow [this post](https://guides.cocoapods.org/making/making-a-cocoapod.html) and [this checklist](https://github.com/thellimist/EZPods).

**Comment by SwiftStudies** [comment]
Hi there. I kind of feel CocoaPods is the past... but feel free to create  a pull

**Comment by iamjono** [comment]
Not sure why you'd use CocoaPods when you have SPM?

**Comment by lfarah** [comment]
CocoaPods measures a ton of information and creates a quality index. We at [AwesomeiOS](https://github.com/vsouza/awesome-ios/) will start using this quality index in a way to filter projects.

By being compatible with CocoaPods, your library gets [Cocoadocs](cocoadocs.org) for free too, so you'll have a better documentation visualization.

CocoaPods is still used by a lot of developers worldwide, so by enabling this you're helping them.

SPM is still not used by a large number of people. Even yesterday at an iOS podcast the speakers were talking about why they aren't still using SPM

**Comment by iamjono** [comment]
I agree Lucas - makes sense for front end projects, but not for server side targets...

**Comment by lfarah** [comment]
Oh, I thought I could use SmarkDown on iOS.... Didn't know it was only a server side lib

**Comment by FranklinYu** [comment]
Not just iOS, application on macOS also requires Cocoa, which doesn't work with SPM currently.
